packages/minuteman/build

```diff
@@ -16,7 +16,9 @@ cat <<EOF > $service
 Description=Layer 4 Load Balancer: DC/OS Layer 4 Load Balancing Service
 After=dcos-gen-resolvconf.service
 After=dcos-epmd.service
+After=dcos-lashup.service
 BindsTo=dcos-epmd.service
+BindsTo=dcos-lashup.service
```

Comment:
Yea... We shouldn't have landed any BindsTo... 

Before/After also give no real guarantee of ordering (Although aren't actively harmful like Wants/Requires and friends)